modules/mod_ginger_base/templates/search-suggestions/search-form.tpl

```diff
@@ -21,9 +21,11 @@ as
 
 <form class="{{ formclass }} {{ extraFormClassess }}" id="search-suggestions__searchform-{{ #identifier }}" role="search" action="{% if context %}/{{ context }}_search{% else %}{% url search %}{% endif %}" method="get">
     <div class="{{ wrapperclass }}">
+        <label for="qs">Zoeken</label>
```

Comment:
Even though only used for screen readers, all labels must be in English: `{_ Search _}`.